supertokens_python/recipe/dashboard/__init__.py

```diff
@@ -14,18 +14,20 @@
 
 from __future__ import annotations
 
-from typing import Optional, Callable
+from typing import TYPE_CHECKING, Callable, Optional, Union
 
-from supertokens_python import AppInfo, RecipeModule
-from supertokens_python.recipe.dashboard.utils import InputOverrideConfig
-
-from .recipe import DashboardRecipe
+if TYPE_CHECKING:
+    from supertokens_python import AppInfo, RecipeModule
+    from supertokens_python.recipe.dashboard.utils import InputOverrideConfig
 
 
 def init(
-    api_key: str,
+    api_key: Union[str, None] = None,
     override: Optional[InputOverrideConfig] = None,
 ) -> Callable[[AppInfo], RecipeModule]:
+    # Global import for the following was avoided because of circular import errors
+    from .recipe import DashboardRecipe
```

Comment:
but there is no more an issue of circular dependency right? Cause we are not doing auto init of dashboard recipe. So remove this comment and make it a regular global import?

Reply:
ok